index.html

```diff
@@ -1,246 +1,283 @@
-                Na OLX.pl czeka na Ciebie praca biurowa, mieszkania, pokoje, samochody. Jeśli chcesz 
-                coś sprzedać - w prosty sposób dodasz bezpłatne ogłoszenia. Chcesz coś kupić - tutaj
-                 znajdziesz ciekawe okazje, taniej niż w sklepie. Sprzedawaj po sąsiedzku na OLX.pl </p>
-                <br>
-                <p>Dołącz do nas:</p>
+      <div class="adv">
+        <img src="ikony/olx.png" alt="" class="iconolx" width="150px" />
+        <p>
+          OLX.pl to darmowe ogłoszenia lokalne w kategoriach: Moda, Zwierzęta,
+          Dla Dzieci, Sport i Hobby, Muzyka i Edukacja. Szybko znajdziesz tu
+          ciekawe ogłoszenia i łatwo skontaktujesz się z ogłoszeniodawcą. Na
+          OLX.pl czeka na Ciebie praca biurowa, mieszkania, pokoje, samochody.
+          Jeśli chcesz coś sprzedać - w prosty sposób dodasz bezpłatne
+          ogłoszenia. Chcesz coś kupić - tutaj znajdziesz ciekawe okazje, taniej
+          niż w sklepie. Sprzedawaj po sąsiedzku na OLX.pl
+        </p>
+        <br />
+        <p>Dołącz do nas:</p>
 
-            <a href="facebook"><img src="ikony/facebook.png" alt="facebook"></a>
-            <a href="instagram"><img src="ikony/instagram.png" alt="instagram"></a>
-            <a href="youtube"><img src="ikony/youtube.png" alt="youtube"></a>
-            <a href="linkedin"><img src="ikony/linkedin.png" alt="linkedin"></a>
-            <a href="pinterest"><img src="ikony/pinterest.png" alt="pinterest"></a>
-        </div>
-
-        <div class="mainCategory">
-            <img src="ikony/note.png" alt="" height="100px">
-            <p><b>Kategorie główne:</b><a href="">Antyki i Kolekcje, </a>
-            <a href="">Motoryzacja, </a><a href="">Nieruchomości, </a><a href="">Praca, </a>
-            <a href="">Dom i Ogród, </a><a href="">Elektronika, </a><a href="">Moda, </a><a href="">Rolnictwo, </a>
-            <a href="">Zwierzęta, </a><a href="">Dla Dzieci, </a><a href="">Sport i Hobby, </a>
-            <a href="">Muzyka i Edukacja, </a><a href="">Zdrowie i Uroda, </a><a href="">Usługi, </a>
-            <a href="">Noclegi, </a><a href="">Wypożyczalnia, </a><a href="">Dla Firm, </a>
-            <a href="">Outlet, </a><a href="">Oddam z darmo</a></p>
-        </div>
+        <a href="facebook"><img src="ikony/facebook.png" alt="facebook" /></a>
+        <a href="instagram"
+          ><img src="ikony/instagram.png" alt="instagram"
+        /></a>
+        <a href="youtube"><img src="ikony/youtube.png" alt="youtube" /></a>
+        <a href="linkedin"><img src="ikony/linkedin.png" alt="linkedin" /></a>
+        <a href="pinterest"
+          ><img src="ikony/pinterest.png" alt="pinterest"
+        /></a>
+      </div>
 
-        <hr>
+      <div class="mainCategory">
+        <img src="ikony/note.png" alt="" height="100px" />
+        <p>
+          <b>Kategorie główne:</b> <a href="">Antyki i Kolekcje, </a>
+          <a href="">Motoryzacja, </a><a href="">Nieruchomości, </a
```

Comment:
Każde a powinno być w nowej linii coś nie 
Zastosuj to rozwiązanie aby porządkować kod
https://blog.yogeshchavan.dev/automatically-format-code-on-file-save-in-visual-studio-code-using-prettier